.github/workflows/ci.yml

```diff
@@ -2,18 +2,24 @@ name: CI
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
   pull_request:
-    branches: [ master ]
+    branches: [master]
 
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
 
+      - name: nvm install
+        shell: bash -l {0}
```

Comment:
Can we put this in defaults to align to other repos?

```
defaults:
  run:
    shell: bash -l {0}
```

Reply:
Given its only used and needed in one place I thought it was not worth it as the reason we do it in other repos is we need it for multiple steps 😄

Reply:
Forget the above looks like we might need it so will change it